docs/schema_registry_tutorial.rst

```diff
@@ -0,0 +1,505 @@
+.. _schema_registry_tutorial:
+
+Confluent Schema Registry Tutorial
+==================================
+
+This tutorial provides an example on how to use |sr-long|.
+
+**Contents**
+
+.. contents::
```

Comment:
Nowadays, we're removing these `.. contents::` blocks, because of the cool, new floating in-page nav thingie.

Reply:
Ok, removed